csv format changes

cg-cnu opened the pull request:
* Changed delimiter from `;` to `,`
* Made the changes in the readme file as well
* Added header info as the first item in the csv file

Thanks! 🙂

daz commented on sensor_database_detailed.csv:
```diff
@@ -1,457 +1,458 @@
+Apple,iPhone 5,"1/3.2"" (~ 4.54 × 3.42 mm),4.54,3.42,3264,2448
+Apple,iPhone 5S,"1/3"" (~ 4.89 × 3.67 mm),4.89,3.67,3264,2448
+Apple,iPhone 6,"1/3"" (~ 4.89 × 3.67 mm),4.89,3.67,3264,2448
+Apple,iPhone 6 Plus,"1/3"" (~ 4.89 × 3.67 mm),4.89,3.67,3264,2448
```
Bad formatting on these iPhone lines. Need an extra `"` after `mm)`

cg-cnu commented:
Thanks for the inputs @daz 
* Fixed the quotes in the Apple,iPhone and RaspberryPi 
* Realized that vscode replaces does only 10,000 items at a time. Fixed the rest this time.
* Fixed a typo in the cameraMaker header title.

cg-cnu commented:
👋 Just a friendly ping! 🙂

rigakevin commented:
You can add CANON EOS 2000D in the database.

The line to add is : Canon;Canon EOS 2000D;22.3
